src/library/browse/browsethread.cpp

```diff
@@ -248,16 +249,20 @@ void BrowseThread::populateModel() {
             item->setData(location, Qt::UserRole);
             row_data.insert(COLUMN_NATIVELOCATION, item);
 
-            QDateTime modifiedTime = pTrack->getFileInfo().fileLastModified().toLocalTime();
-            item = new QStandardItem(modifiedTime.toString(Qt::DefaultLocaleShortDate));
+            const auto fileLastModified =
+                    pTrack->getFileInfo().fileLastModified();
+            item = new QStandardItem(
+                    mixxx::displayDateTime(fileLastModified));
```

Comment:
```suggestion
                    mixxx::displayDateTime(fileLastModified.toLocalTime()));
```
Or is this not necessary?

Reply:
I checked it. No, not necessary. Qt::DefaultLocaleShortDate already performs the conversion into the local time zone. I will add a comment for the function.

Reply:
I will rename the function: displayLocalDateTime()